src/main/java/Parser.java

```diff
@@ -0,0 +1,180 @@
+import java.io.IOException;
+import java.time.LocalDate;
+
+/**
+ * Parser handles the user commands.
+ */
+public class Parser {
+
+    protected Storage storage;
+    protected TaskList tasks;
+    protected Ui ui;
+    protected String command;
+    protected int index;
+    protected int findSlash;
+    protected int taskIdentifier;
+
+    /**
+     * Creates a new instance of a Parser to make sense of user command.
+     * @param storage Storage of tasks on hard disk.
+     * @param tasks List of user's tasks.
+     * @param ui Ui for user interaction.
+     */
+    public Parser(Storage storage, TaskList tasks, Ui ui) {
+        this.storage = storage;
+        this.tasks = tasks;
+        this.ui = ui;
+    }
+
+    /**
+     * Handles the ToDo command.
+     * @throws IOException On file error when adding the ToDo task
+     * to the file in the hard disk.
+     */
+    public void handleToDo() throws IOException {
+        String descriptionTask = command.substring(index + 1);
```

Comment:
"Avoid magic numbers" https://nus-cs2103-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers

Might want to use a named constant for this? Might want to use this style: https://github.com/se-edu/addressbook-level2/blob/master/src/seedu/addressbook/ui/TextUi.java#L25